src/components/Card/index.js

```diff
@@ -0,0 +1,7 @@
+import Item from './Item'
```

Comment:
So Based on the above this is where your actual component will live, so put the code from `Item` into here…